### PR TITLE
Revert #47 and update to 3.12.1

### PR DESCRIPTION
**Pull request by jjhelmus**
Revert switch to cmake done in PR #47 
Update to version 3.12.1

### CODE REVIEW

**Comment by jjhelmus** [comment]
Looks like GitHub is a bit unstable at the moment which is causing some fetches and curl downloads to fail.

File: recipe/build.sh [hunk 1]
```diff
@@ -21,16 +21,22 @@ fi
...
+autoreconf -i
+automake --add-missing
 
+./configure --prefix="${PREFIX}" \
+            --build=${HOST}      \
+            --host=${HOST}       \
+            --with-pic           \
+            --with-zlib          \
+            --enable-shared      \
+            --enable-static      \
```
Comment (xhochy) [inline recipe/build.sh:37]:
```suggestion
```

File: recipe/meta.yaml [hunk 2]
```diff
@@ -43,6 +48,7 @@ requirements:
 test:
   commands:
     - protoc --help
+    - test -f ${PREFIX}/lib/libprotobuf.a  # [not win]
```
Comment (xhochy) [inline recipe/meta.yaml:51]:
```suggestion
```

**Comment by xhochy** [comment]
@conda-forge-admin please restart ci

**Pull request merged**